examples/kubecon-na-22-demo/README.md

```diff
@@ -5,13 +5,20 @@ This demonstration is based on the [Observability Example](../observability) and
 ![img.png](assets/big-picture.png)
 
 ## Prepare Secret for Slack Notification
-> kubectl create secret generic slack-notification --from-literal=SECURE_DATA='{"slack_hook":"HOOK","text":"Deployed PodTatoHead Application"}' -n podtato-kubectl -oyaml --dry-run > base/slack-secret.yaml
+As a first step, create an incoming webhook according to the instructions
+https://api.slack.com/messaging/webhooks
+
+Afterwards create a secret with the created webhook
+> kubectl create secret generic slack-notification --from-literal=SECURE_DATA='{"slack_hook":<YOUR_HOOK>,"text":"Deployed PodTatoHead Application"}' -n podtato-kubectl -oyaml --dry-run=client > base/slack-secret.yaml
 
 ## Deploy the Observability Part
 > make install
 
 ## Port-Forward Grafana
-> make port-forward-grafana
+> make port-forward-grafana GRAFANA_PORT_FORWARD=3000
```

Comment:
Why do I have to specify the default port?

Reply:
defaultly we do not have, but it is possible to override the 3000 default port with a custom one. A user may have something already running on this port (as for example I did), so it's nice to have an option when you can override it

Reply:
ah sorry, I left the port here in the README by mistake, removing it, but the functionality to override the default port stays